app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -54,6 +63,7 @@ def new
     end
 
     def confirm
+      increment_mfa_selection_attempt_count(:webauthn)
```

Comment:
We'll want to use the constant here as well, and incorporate `'webauthn_platform'`.

Maybe we want a helper method to avoid noisy `if` `else` ?

```rb
def webauthn_auth_method
  if @platform_authenticator
    TwoFactorAuthenticatable::AuthMethod::WEBAUTHN_PLATFORM
  else
    TwoFactorAuthenticatable::AuthMethod::WEBAUTHN
  end
end
```

Reply:
That works great - also for sms/voice.